packages/vc-export/src/vc-js/suites/KiltAbstractSuite.ts

```diff
@@ -83,7 +83,7 @@ export abstract class KiltAbstractSuite extends suites.LinkedDataProof {
   }): Promise<boolean> {
     const { proof } = options
     const compact = await this.compactProof(proof, options)
-    const type = compact['@type'] || compact.type
+    const type = compact['@type'] ?? compact.type
```

Comment:
In all cases of amorphous JSON we get a type which can be a numeric value of `0`, and should use nullish coalescence